recipe/meta.yaml

```
@@ -44,41 +47,43 @@ outputs:
       script: 'python -m pip install . --no-deps --ignore-installed -vvv '
     requirements:
       host:
-        - python
+        - python {{ python_min }}
         - pip
         - flit-core
       run:
-        - python ~=3.9
+        - python >={{ python_min }}
         - alembic ~=1.2
         - archive-path ~=0.4.2
-        - aio-pika ~=6.6
-        - circus ~=0.18.0
+        - asyncssh ~=2.19.0
+        - circus ~=0.19.0
         - click-spinner ~=0.1.8
         - click ~=8.1
-        - disk-objectstore ~=1.1
+        - disk-objectstore ~=1.3.0
         - docstring_parser
         - get-annotations ~=0.1
-        - jedi <0.19
         - python-graphviz ~=0.19
+        - plumpy ~=0.25.0
         - ipython >=7
+        - jedi <0.19
         - jinja2 ~=3.0
-        - kiwipy[rmq] ~=0.7.7
+        - kiwipy[rmq] ~=0.8.4
         - importlib-metadata ~=6.0
         - numpy ~=1.21
-        - paramiko ~=2.7,>=2.7.2
-        - plumpy ~=0.21.6
-        - pgsu ~=0.2.1
+        - paramiko ~=3.0
+        - pgsu ~=0.3.0
         - psutil ~=5.6
-        - psycopg2-binary ~=2.8
+        - psycopg[binary] ~=3.0
         - pydantic ~=2.4
         - pytz ~=2021.1
         - pyyaml ~=6.0
         - requests ~=2.0
         - sqlalchemy ~=2.0
-        - tabulate >=0.8.0,<0.10.0
+        - tabulate <0.10.0,>=0.8.0
         - tqdm ~=4.45
+        - typing-extensions ~=4.0
         - upf_to_json ~=0.9.2
         - wrapt ~=1.11
+        - chardet ~=5.2.0
```

This is only needed on windows, I think this should work

```suggestion
        - chardet ~=5.2.0 [win]
```

---

the linter complained because we use noarch, so I did not wanted to add it halfbaked

---

Ah, okay, fair enough